src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1020,6 +1027,25 @@ std::vector< peer_info > RaftReplDev::get_replication_status() const {
     return pi;
 }
 
+std::set< replica_id_t > RaftReplDev::get_active_peers() const {
+    auto repl_status = get_replication_status();
+    std::set< replica_id_t > res;
+    auto my_committed_idx = m_commit_upto_lsn.load();
+    uint64_t lagThreshold = my_committed_idx > HS_DYNAMIC_CONFIG(consensus.laggy_threshold)
```

Comment:
nit: since HomeStore uses `my_local_var` name format, can we change to `lag_threshold`?